ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/SchemaInfo.java

```diff
@@ -36,6 +37,23 @@
   private final SqlBaseType type;
   private final ImmutableList<FieldInfo> fields;
   private final SchemaInfo memberSchema;
+  private final Map<String, Object> fieldProperties;
+
+  @JsonCreator
+  public SchemaInfo(
```

Comment:
Can you also update `ConsoleTest#shouldPrintTypesList` to include a field that is a decimal? that will give us more confidence in what the user ends up seeing.

Reply:
Ah couldn't find the right test on the other end, nice